src/modules/addRemove.js

```diff
@@ -205,8 +206,3 @@ document.addEventListener('click', (event) => {
     // listRefresher();
   }
 });
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your commit message capitalized to make your pull request more professional 👍